plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java

```diff
@@ -177,7 +180,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         if (offline) {
             getLog().debug("Executing offline CLI scripts");
             try (StandardOutputStream out = StandardOutputStream.parse(stdout, false)) {
-                final int exitCode = offlineCLIExecutor.execute(jbossHome, getCommands(), getLog(), out, systemProperties);
+
+                String[] opts = javaOpts.split("[\\n\\s]+");
```

Comment:
This here is so we can support properties passes in multiple lines as well as just separated by space same as surefire does it